src/sagemaker/feature_store/feature_group.py

```diff
@@ -457,6 +457,12 @@ def create(
             online_store_kms_key_id (str): KMS key id for online store.
             enable_online_store (bool): whether to enable online store or not.
             offline_store_kms_key_id (str): KMS key id for offline store.
+                If KMS encryption key is not specified, by default we encrypt all data at
+                rest using AWS KMS key. By defining your bucket-level key for SSE, you can
+                reduce AWS KMS requests costs by up to 99 percent.
+                For more information, see the S3 documentation for
+                `Bucket Key
+                <https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-key.html>`_.
```

Comment:
AWS KMS key ID for offline store.
If a KMS encryption key is not specified, SageMaker encrypts all data at
rest using the default AWS KMS key. By defining your bucket-level key for SSE, you can
reduce the cost of AWS KMS requests.
For more information, see
`Bucket Key
<https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-key.html>`_
in the *Amazon S3 User Guide*.

Reply:
(You should not use any marketing words in the sdk docstrings.

Reply:
Revised according to your comments, thanks!